.github/workflows/ci-actions.yml

```diff
@@ -83,7 +83,7 @@ jobs:
           key: q2maven-${{ steps.get-date.outputs.date }}
       - name: Build
         run: |
-          mvn -e -B -DskipTests -DskipITs -Dno-format clean install
+          mvn -T1C -e -B -DskipTests -DskipITs -Dinvoker.skip -Dno-format clean install
```

Comment:
Ah nice, I wanted to experiment with that.

Reply:
Too bad GH only gives us two cores so it's still rather slow (> 10 mins).